### ISSUE TITLE
No buffer named package.namespace

### ISSUE THREAD

**Issue by kostafey** [clojure-emacs/cider#1294]
After last update CIDER output the following message on `(cider-pprint-eval-last-sexp)` call:

`nrepl-send-request: No buffer named <package-name.file-name>`

cider-20150901.1605

UPD: trace

```
Debugger entered--Lisp error: (error "No buffer named package.namespace")
  nrepl-next-request-id("package.namespace")
  nrepl-send-request(("pprint" "true" "ns" 80 "op" "eval" "session" nil "code" "(+ 1 1)") #[257 "\306\307\"\306\310\"\306\311\"\306\312\"\306\313\"\306\314\"\306\315\"\306\316\"\317\300!\203=r\300q\210\203<\320\321!\204<)\203N\301\205\313\301\300	\"\202\313\203^\302\205\313\302\300\"\202\313\211\203m\302\205\313\302\300\"\202\313\203}\303\205\313\303\300\"\202\313\205\313\322\235\203\213\323\324!\210\325\235\203\231\305\206\227 \210\326\235\203\243\323\327!\210\330\235\203\255\331\300!\210\332\235\205\313\333\334 \"!#\210\335 \"\210\304\205\313\304\300!\207" [#<buffer *cider-result*> nil #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] nil nil nrepl-dict-get "value" "ns" "out" "err" "status" "id" "session" "pprint-out" buffer-live-p derived-mode-p clojure-mode "interrupted" message "Evaluation interrupted." "eval-error" "namespace-not-found" "Namespace not found." "need-input" cider-need-input "done" puthash gethash remhash cider-buffer-ns nrepl-err-handler nrepl-pending-requests nrepl-completed-requests] 14 "\n\n(fn RESPONSE)"] "package.namespace")
  nrepl-request:pprint-eval("(+ 1 1)" #[257 "\306\307\"\306\310\"\306\311\"\306\312\"\306\313\"\306\314\"\306\315\"\306\316\"\317\300!\203=r\300q\210\203<\320\321!\204<)\203N\301\205\313\301\300	\"\202\313\203^\302\205\313\302\300\"\202\313\211\203m\302\205\313\302\300\"\202\313\203}\303\205\313\303\300\"\202\313\205\313\322\235\203\213\323\324!\210\325\235\203\231\305\206\227 \210\326\235\203\243\323\327!\210\330\235\203\255\331\300!\210\332\235\205\313\333\334 \"!#\210\335 \"\210\304\205\313\304\300!\207" [#<buffer *cider-result*> nil #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] nil nil nrepl-dict-get "value" "ns" "out" "err" "status" "id" "session" "pprint-out" buffer-live-p derived-mode-p clojure-mode "interrupted" message "Evaluation interrupted." "eval-error" "namespace-not-found" "Namespace not found." "need-input" cider-need-input "done" puthash gethash remhash cider-buffer-ns nrepl-err-handler nrepl-pending-requests nrepl-completed-requests] 14 "\n\n(fn RESPONSE)"] "package.namespace" nil 80)
  cider-interactive-pprint-eval("(+ 1 1)" #[257 "\306\307\"\306\310\"\306\311\"\306\312\"\306\313\"\306\314\"\306\315\"\306\316\"\317\300!\203=r\300q\210\203<\320\321!\204<)\203N\301\205\313\301\300	\"\202\313\203^\302\205\313\302\300\"\202\313\211\203m\302\205\313\302\300\"\202\313\203}\303\205\313\303\300\"\202\313\205\313\322\235\203\213\323\324!\210\325\235\203\231\305\206\227 \210\326\235\203\243\323\327!\210\330\235\203\255\331\300!\210\332\235\205\313\333\334 \"!#\210\335 \"\210\304\205\313\304\300!\207" [#<buffer *cider-result*> nil #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] #[514 "\300\"\207" [cider-emit-into-popup-buffer] 5 "\n\n(fn BUFFER STR)"] nil nil nrepl-dict-get "value" "ns" "out" "err" "status" "id" "session" "pprint-out" buffer-live-p derived-mode-p clojure-mode "interrupted" message "Evaluation interrupted." "eval-error" "namespace-not-found" "Namespace not found." "need-input" cider-need-input "done" puthash gethash remhash cider-buffer-ns nrepl-err-handler nrepl-pending-requests nrepl-completed-requests] 14 "\n\n(fn RESPONSE)"] 80)
  cider--pprint-eval-form("(+ 1 1)")
  cider-pprint-eval-last-sexp()
  call-interactively(cider-pprint-eval-last-sexp nil nil)
  command-execute(cider-pprint-eval-last-sexp)

```

**Comment by cichli** [comment]
Thanks for the report - this was fixed in clojure-emacs/cider@3e349682323e5bec9e562c8c6971a87e9543da72 - will you try again after updating?

**Comment by kostafey** [comment]
Now it's ok. Thank you!
